README.md

```diff
@@ -159,15 +159,28 @@ Important Notes:
 
 ## Error States
 
+### Error 400: Bad Request, no primary key
+If a WAL record for a table that does not have a primary key is passed through `realtime.apply_rls`, an error is returned
+
+Ex:
+```sql
+(
+    null,                            -- wal
+    null,                            -- is_rls_enabled
+    [...],                              -- users,
+    array['Error 400: Bad Request, no primary key'] -- errors
```

Comment:
@olirice just something to think about but what if a table has replica identity set to full? doesn't that technically mean the entire row becomes the key?

Reply:
kinda, but there's no enforcement of uniqueness (afaik). If there is no primary key we also lose the guarantee that the RLS checking prepared statement will hit an index.

Since disabling a primary key could be an optimization to improve insert throughput (high volume/large tables) it would be very risky to allow